featuretools/tests/computational_backend/test_feature_set_calculator.py

```diff
@@ -746,3 +749,17 @@ def get_function(self):
     assert all(fm[f5.get_name()].sort_index() == value_sum)
     assert all(fm[f6.get_name()].sort_index() == value_sum)
     assert all(fm[f7.get_name()].sort_index() == value_sum)
+
+
+def test_returns_order_of_instance_ids(es):
+    feature_set = FeatureSet([ft.Feature(es['customers']['age'])])
+    calculator = FeatureSetCalculator(es,
```

Comment:
this works to make sure `FeatureSetCalculator` returns the input order, but I don't believe we have a test to confirm `calculate_feature_matrix` does that. 

we don't need to address that in this PR, just noting it for future reference